src/components/SearchPage/SearchPage.js

```diff
@@ -14,7 +15,7 @@ const SearchPage = ({setNewPost}) => {
 
     const { loading, error, data } = useQuery(gql.GET_ALL_USERS);
 
-    if (loading) return <p>Loading...</p>;
+    if (loading) return <Loading loading={loading} />;
```

Comment:
I really like the new use of the loading component, maybe we can make it match up with the rest of the color scheme